src/pages/plugins/index.js

```diff
@@ -95,21 +101,28 @@ function PluginFooter({ keywords = [] }) {
 }
 
 function PluginComponent({ plugin }) {
+  const deprecatedCard = plugin.deprecatedMessage ? true : false;
```

Comment:
You can implement this as `isDeprecated` on the class, like you implemented `deprecatedMessage`

Reply:
Set the isDeprecated in the class get method